catalogue_graph/scripts/build.sh

```diff
@@ -57,12 +63,18 @@ function build_zip() {( set -e
     mkdir -p target/tmp
 
     cp -r src/* target/tmp
+    
+    # Use UV to install dependencies
+    uv export --no-dev --format requirements-txt > target/tmp/requirements.txt
```

Comment:
Can the installation be done with `uv` alone, avoiding generating an intermediate `requirements.txt`?